spec/controller/demographics/districts_controller_spec.rb

```diff
@@ -0,0 +1,112 @@
+require 'rails_helper'
+
+RSpec.describe Api::V1::Demographics::DistrictsController, type: :controller do
+  describe "GET show" do
+    it "returns a specific districts demographic information" do
+      sy = SchoolYear.create(years: "2015-2016")
```

Comment:
The same comment as the one above. Be careful using short variable names. I'm even more inclined to suggest you type these out in full since I tend to look at tests as documentation. If I have to figure out what `d` means when I'm debugging something, that may make me a little miffed.

I know there will be people out there that will argue against this position. But since autocomplete within any reasonable text editor makes this sort of typing relatively painless, I just tend to aim toward being more verbose instead of less.